src/System.Management.Automation/engine/remoting/commands/NewPSSessionConfigurationFile.cs

```diff
@@ -967,7 +967,7 @@ protected override void ProcessRecord()
                                 ThrowTerminatingError(e.ErrorRecord);
                             }
 
-                            if ((hashtable[ConfigFileConstants.FunctionValueToken] as ScriptBlock) == null)
+                            if (!(hashtable[ConfigFileConstants.FunctionValueToken] is ScriptBlock))
```

Comment:
```suggestion
                            if (hashtable[ConfigFileConstants.FunctionValueToken] is not ScriptBlock)
```

Reply:
@iSazonov This is a good example how `is not` can make code more readable.

Nevertheless, do you not think it would be preferable to isolate C# 9.0 changes such as `is not` in a seperate PR?

Reply:
We discussed this and this was approved. (It is incredible that this will be canceled.)